tools/SchemaManager/SchemaManager.csproj

```diff
@@ -11,13 +11,18 @@
 
   <ItemGroup>
     <PackageReference Include="Ensure.That" Version="9.2.0" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.5.0" />
     <PackageReference Include="Microsoft.SqlServer.SqlManagementObjects" Version="161.41011.9" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Polly" Version="7.2.1" />
     <PackageReference Include="System.CommandLine.Experimental" Version="0.2.0-alpha.19174.3" />
     <PackageReference Include="System.CommandLine.Rendering" Version="0.2.0-alpha.19174.3" />
     <PackageReference Include="Microsoft.Data.SqlClient" Version="2.0.1" />
-    <PackageReference Include="Microsoft.Health.SqlServer" Version="1.0.0-master-20200826-1" />
+    <PackageReference Include="Microsoft.Health.SqlServer" Version="1.0.0-master-20200827-1" />
```

Comment:
We probably don't need this reference if it's being changed to a project reference.

Reply:
Yep, will update.